BraveWallet/WalletConstants.swift

```diff
@@ -47,6 +47,11 @@ struct WalletConstants {
   
   /// The link for users to learn more about Solana SPL token account creation in transaction confirmation screen
   static let splTokenAccountCreationLink = URL(string: "https://support.brave.com/hc/en-us/articles/5546517853325")!
+  
+  /// Supported networks for buying via Wyre
+  static let supportedBuyWithWyreNetworkChainIds: [String] = [
+    BraveWallet.MainnetChainId
```

Comment:
minor nit: can we reference the #5781 issue here for why Polygon and Avalanche were removed in this PR?